packages/styled-components/src/sheet/test/WindowedTag.test.js

```diff
@@ -0,0 +1,192 @@
+// @flow
+
+import { VirtualTag } from '../Tag';
+import { WindowedTag } from '../WindowedTag';
+import { DefaultGroupedTag } from '../GroupedTag';
+
+describe('WindowedTag acts like a tag', () => {
+  let virtualTag;
+  let tag;
+  beforeEach(() => {
+    virtualTag = new VirtualTag();
+    tag = new WindowedTag(virtualTag);
+  });
+
+  /*
+   * Copy/Pasted from Tag.test.js since it requires different setup than the tags tested there
```

Comment:
I'd love to not duplicate these tests in this file (and with the GroupedTag tests), but I wanted to make sure that the `WindowedTag` passes the same tests we're using for the `Tag`(s), but I couldn't think of a good way to utilize the setup in Tag.test.js :(